src/intTest/resources/patient/parameters.json

```diff
@@ -0,0 +1,104 @@
+{
+  "parameter": [
+    {
+      "name": "patient",
+      "resource": {
+        "address": [
+          {
+            "line": [
+              "current house name",
+              "current number or road name",
+              "current locality",
+              "current town",
+              "current county"
+            ],
+            "postalCode": "SO15 2GB",
+            "text": "house name, number or road name, locality, town, country",
```

Comment:
Is this "text" something generated by HAPI? We won't be using it.

Reply:
I've used an example from hl7.org:
https://www.hl7.org/fhir/patient-example.json.html

I'll update address because it's using 'text'

Reply:
We shouldn't be using 'text' right? So these properties can just be removed here. We should not include in out test data properties we don't actually use. They'll make their way into our examples, or a use will look to the tests for how to send requests and get confused.

Reply:
sure, I'll remove it. Just need to change implementation ;)

Reply:
We'll do integration tests using bartek's framework in the future - will fix forward